service/src/main/java/bio/terra/workspace/service/policy/TpsApiDispatch.java

```diff
@@ -193,6 +201,26 @@ public List<String> listValidDataCenter(UUID workspaceId, String platform) {
     return new ArrayList<>();
   }
 
+  @Traced
+  @Cacheable(cacheNames = "regionInfo", sync = true)
```

Comment:
Curious if the tests would also get cached data. Seems like they would. It shouldn't really matter since the data doesn't change that much.

Reply:
I wonder if this is a premature optimization. Do we know that we have a performance or load issue? Is WSM the right place to put caching or should it be in TPS? A TPS cache would have the advantage of knowing when to invalidate.